api/v1beta1/conditions.go

```diff
@@ -111,4 +111,7 @@ const (
 
 	// KeystoneServiceOSUserReadyErrorMessage
 	KeystoneServiceOSUserReadyErrorMessage = "Keystone Service user error occured %s"
+
+	// KeystoneNetworkAttachmentsReadyErrorMessage
+	KeystoneNetworkAttachmentsReadyErrorMessage = "NetworkAttachments error occurred; not all pods have interfaces with IPs as configured in NetworkAttachments: %s"
```

Comment:
lets use the common one Kamil added to lib-common https://github.com/openstack-k8s-operators/lib-common/blob/main/modules/common/condition/conditions.go#L272